phc-rails/app/controllers/api/v1/event_registrations_controller.rb

```diff
@@ -31,13 +31,14 @@ def create
       (event_reg.services ||= []) << Service.new(name: service, status:status)
     end
 
-    render :json => { status: (event_reg.save ? "Success" : "Failure") }
+    status = event_reg.save ? "Success" : "Failure"
```

Comment:
You're going to have to talk to @altonzheng about this (I can't take credit for this code).